src/components/DpAutocomplete/DpAutocomplete.vue

```diff
@@ -41,6 +71,15 @@
         @mousedown.stop.prevent="focusInput"
         @keydown.stop="runSpecialKeys"
         @focusout="isInputFocused = false" />
+      <dp-button
+        v-if="searchButton && !isPlaceholderVisible"
```

Comment:
i am a bit confused about which button appears when, we have two `dp-button` components with the search icon and only their conditions differ. Maybe it could be refactored like this to remove the duplication:
```
<dp-button
  v-if="(isPlaceholderVisible || (searchButton && !isPlaceholderVisible))"
  ....other attributes...
  :class="isPlaceholderVisible ? 'border-2 border-l' : 'border-2 !border-l-1'"
 >
```

Reply:
Thanks, you're absolutely right e36907913e0bfea1d19584baab4f398ef40e2f9a